modules/digitaluniverse/rendering/renderableplanescloud.cpp

```diff
@@ -68,13 +68,6 @@ namespace {
         BlendModeAdditive
     };
 
-    constexpr openspace::properties::Property::PropertyInfo TransparencyInfo = {
-        "Transparency",
-        "Transparency",
-        "This value is a multiplicative factor that is applied to the transparency of "
-        "all points."
-    };
-
     constexpr openspace::properties::Property::PropertyInfo ScaleFactorInfo = {
```

Comment:
Probably remove this Property if it is commented out and unused. Also the documentation for it is wrong.

Reply:
This is another class than the one where the property was commented out, and in this one it is actually used ^^' But I'll make sure to update the description. Good find!

Reply:
Actually, looks like the description wasn't wrong after all?

Reply:
Ok, yes, I must've put this comment to the wrong file that also had the ScaleFactorInfo. If you found it that is good, I remember the property description was the same as TextColor and then the _scaleFactor property was commented out everywhere in the file.

Reply:
Ah I see. Well, made sure to remove the property in that file so the matter is solved then ^^